test/parallel/test-url-pathtofileurl.js

```diff
@@ -38,8 +38,8 @@ const url = require('url');
     });
   } else {
     // UNC paths on posix are considered a single path that has backslashes:
-    const fileURL = url.pathToFileURL('\\\\nas\\share\\path.txt').href;
-    assert.match(fileURL, /file:\/\/.+%5C%5Cnas%5Cshare%5Cpath\.txt$/);
+    const fileHref = url.pathToFileURL('\\\\nas\\share\\path.txt').href;
+    assert.match(fileHref, /file:\/\/.+%5C%5Cnas%5Cshare%5Cpath\.txt$/);
```

Comment:
I don’t think you intended to change this file?

Perhaps we should just change the function itself without renaming it, so that we don’t create a large diff?